### ISSUE TITLE
Fail to compile (Julia 1.5.1)

### ISSUE THREAD

**Issue by fastflair** [SciML/DiffEqFlux.jl#399]
I'm obtaining this error with compiling the Flux library (Ubuntu 18.04 OS, CUDA driver version = 10.1).
┌ Info: Precompiling Flux [587475ba-b771-5e3f-ad9e-33799f191a9c]
└ @ Base loading.jl:1278
ERROR: LoadError: LoadError: LoadError: UndefVarError: AddrSpacePtr not defined
Stacktrace:
 [1] getproperty(::Module, ::Symbol) at ./Base.jl:26
 [2] top-level scope at /opt/julia/latest/packages/packages/CUDAnative/ierw8/src/device/cuda/wmma.jl:56
 [3] include(::Function, ::Module, ::String) at ./Base.jl:380
 [4] include at ./Base.jl:368 [inlined]
 [5] include(::String) at /opt/julia/latest/packages/packages/CUDAnative/ierw8/src/CUDAnative.jl:1
 [6] top-level scope at /opt/julia/latest/packages/packages/CUDAnative/ierw8/src/device/cuda.jl:14
 [7] include(::Function, ::Module, ::String) at ./Base.jl:380
 [8] include at ./Base.jl:368 [inlined]
 [9] include(::String) at /opt/julia/latest/packages/packages/CUDAnative/ierw8/src/CUDAnative.jl:1
 [10] top-level scope at /opt/julia/latest/packages/packages/CUDAnative/ierw8/src/CUDAnative.jl:70
 [11] include(::Function, ::Module, ::String) at ./Base.jl:380
 [12] include(::Module, ::String) at ./Base.jl:368
 [13] top-level scope at none:2
 [14] eval at ./boot.jl:331 [inlined]
 [15] eval(::Expr) at ./client.jl:467
 [16] top-level scope at ./none:3
in expression starting at /opt/julia/latest/packages/packages/CUDAnative/ierw8/src/device/cuda/wmma.jl:55
in expression starting at /opt/julia/latest/packages/packages/CUDAnative/ierw8/src/device/cuda.jl:14
in expression starting at /opt/julia/latest/packages/packages/CUDAnative/ierw8/src/CUDAnative.jl:70
ERROR: LoadError: Failed to precompile CUDAnative [be33ccc6-a3ff-5ff2-a52e-74243cff1e17] to /opt/julia/latest/packages/compiled/v1.5/CUDAnative/4Zu2W_tPeEh.ji.
Stacktrace:
 [1] error(::String) at ./error.jl:33
 [2] compilecache(::Base.PkgId, ::String) at ./loading.jl:1305
 [3] _require(::Base.PkgId) at ./loading.jl:1030
 [4] require(::Base.PkgId) at ./loading.jl:928
 [5] require(::Module, ::Symbol) at ./loading.jl:923
 [6] include(::Function, ::Module, ::String) at ./Base.jl:380
 [7] include(::Module, ::String) at ./Base.jl:368
 [8] top-level scope at none:2
 [9] eval at ./boot.jl:331 [inlined]
 [10] eval(::Expr) at ./client.jl:467
 [11] top-level scope at ./none:3
in expression starting at /opt/julia/latest/packages/packages/CuArrays/YFdj7/src/CuArrays.jl:3
ERROR: LoadError: Failed to precompile CuArrays [3a865a2d-5b23-5a0f-bc46-62713ec82fae] to /opt/julia/latest/packages/compiled/v1.5/CuArrays/7YFE0_tPeEh.ji.
Stacktrace:
 [1] error(::String) at ./error.jl:33
 [2] compilecache(::Base.PkgId, ::String) at ./loading.jl:1305
 [3] _require(::Base.PkgId) at ./loading.jl:1030
 [4] require(::Base.PkgId) at ./loading.jl:928
 [5] require(::Module, ::Symbol) at ./loading.jl:923
 [6] include(::Function, ::Module, ::String) at ./Base.jl:380
 [7] include(::Module, ::String) at ./Base.jl:368
 [8] top-level scope at none:2
 [9] eval at ./boot.jl:331 [inlined]
 [10] eval(::Expr) at ./client.jl:467
 [11] top-level scope at ./none:3
in expression starting at /opt/julia/latest/packages/packages/Flux/Fj3bt/src/Flux.jl:26

**Comment by avik-pal** [comment]
What are the versions of the libraries (CuArrays/CUDAnative were merged into CUDA.jl)? `st`

**Comment by ChrisRackauckas** [comment]
This looks like an LLVM issue. Did you install the Julia binary by downloading it from https://julialang.org/downloads/ or did you get it from something like `apt`?

**Comment by fastflair** [comment]
I installed it from the https://julialang.org/downloads/ site.  However, I deleted and reinstalled Julia-1.5.1 and am getting different results by only adding the Flux library versus adding multiple libraries at once.  This route is installing the CUDA110, CUDNN_CUDA110, CUTENSOR_CUDA110 artifacts, which didn't occur the first time.  I was able to add all the other libraries and compile after each once succesfully.  The previous results were from adding all the libraries are once versus compiling them individually after adding them one at a time.  The libraries were : using Flux, DifferentialEquations, Optim, DiffEqFlux, Plots, OrdinaryDiffEq, LinearAlgebra, Random
This got me going, though I'm not sure why.  Thank you for the insight!
